src/components/authorize/tests/authorize-integration.test.ts

```diff
@@ -4,9 +4,17 @@ import { sinon } from "../../../../test/utils/test-utils";
 import nock = require("nock");
 import decache from "decache";
 import { HTTP_STATUS_CODES, PATH_NAMES } from "../../../app.constants";
-import { AuthorizeServiceInterface, StartAuthResponse } from "../types";
+import {
+  AuthorizeServiceInterface,
+  JwtServiceInterface,
+  KmsDecryptionServiceInterface,
+  StartAuthResponse,
+} from "../types";
 import { createApiResponse } from "../../../utils/http";
 import { AxiosResponse } from "axios";
+import { createmockclaims, getPrivateKey, getPublicKey } from "./test-data";
+import { JwtService } from "../jwt-service";
+import { createJwt } from "./test-data";
```

Comment:
Sonar just wants you to roll this in with the import a couple of lines up - I guess you already saw, but just in case ^